docs/docs/30-administration/22-backends/20-ssh.md

```diff
@@ -0,0 +1,11 @@
+# SSH backend
+
+The local backend will execute the pipelines using SSH on a remote system without any isolation of any kind.
+
+Since the code run directly in the same context as the agent (same user, same filesystem), a malicious pipeline could 
+be used to access the agent configuration especially the `WOODPECKER_AGENT_SECRET` variable.
+
+It is recommended to use this backend only for private setup where the code and pipeline can be trusted. You shouldn't
+use it for a public facing CI where anyone can submit code or add new repositories.
+
+The backend will use a random directory in $TMPDIR to store the clone code and execute commands.
```

Comment:
That's not the case at the moment.

Reply:
Wasn't that done here: https://github.com/woodpecker-ci/woodpecker/pull/861/commits/2abd0743fed0e955f57cc360b065bc428dcd45eb?

Reply:
Strange, I didn't see that commit, guess there was a mixup in github notications or something, sorry. However, this create the directory on the agent, not on the system where ssh is run.